assets/app/styles/_list.less

```diff
@@ -0,0 +1,22 @@
+.confirm-delete {
+  background: @state-warning-bg;
+  color: @state-warning-text;
+  margin-top: 0px;
+  padding: 10px 25px;
+  position: relative;
+}
+
+.delete-button {
+  color: @gray-light;
+  &:hover {
+  	color: #000;
+  }
+}
+
+.deleting {
```

Comment:
can drop this style if we get rid of the blue message that pops up after confirming deletion